### ISSUE TITLE
question: does proxy to subfolders work?

### ISSUE THREAD

**Issue by Anilkonda** [evertramos/docker-compose-letsencrypt-nginx-proxy-companion#106]
Hi,
Thanks for a great repo. I am using it in the production successfully. I want to know whether this repo supports   forwarding subdirectory requests to different containers.

EG:
www.example.com   --> main container
www.example.com/api --> api container
www.example.com/email --> email container service

how can we acheive this kind of functionality. If it's supported please put that in the docs. I think this feaature very useful.

Thanks.

**Comment by evertramos** [comment]
Hello @Anilkonda,

Thanks for you message and using our repo.

I have tried to add this feature myself a couple times, but once the initial developer does not want to add it and I do understand why... I decided to put another nginx for subfolder for each domain which for me solved the issue and I have full control of how should be redirected and in one of our project nginx plus with jwt authentication as well. So I suggest you use the a new nginx container for that purpose.

Here is an option I have tested and couldn´t get it running as expected:

https://github.com/jwilder/nginx-proxy/pull/1083

Hope you come with the best solution for you case and if you find anything useful to share we will be open to it.

Thanks again!
